src/Tribe/Template.php

```diff
@@ -263,10 +263,10 @@ public function set_template_context_extract( $value = false ) {
 	 *
 	 * @return array
 	 */
-	public function merge_context( $context = array(), $file = null, $name = null ) {
+	public function merge_context($context = [], $file = null, $name = null ) {
```

Comment:
* :no_entry_sign: No space after opening parenthesis is prohibited
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Expected 1 spaces after opening parenthesis; 0 found
* :no_entry_sign: There must be a single space after the first parenthesis
* :no_entry_sign: Short array syntax is not allowed

<sub>hash: c5468ba657774707c38426a5d28dacc4</sub>